compiler/optimizer/LoopVersioner.cpp

```diff
@@ -2904,7 +2904,7 @@ bool TR_LoopVersioner::checkProfiledGuardSuitability(TR_ScratchList<TR::Block> *
             TR::MethodSymbol *method = callSymRef->getSymbol()->castToMethodSymbol();
             TR_ResolvedMethod *owningMethod = callSymRef->getOwningMethod(comp);
             int32_t len = method->getMethod()->classNameLength();
-            char *s = TR::Compiler->cls.classNameToSignature(method->getMethod()->classNameChars(), len, comp);
+            const char *s = TR::Compiler->cls.classNameToSignature(method->getMethod()->classNameChars(), len, comp);
```

Comment:
Strictly speaking, in my previous review I was only asking about places where the result of calling `cls.classNameChars` was assigned to a `char *` variable instead of a `const char *`.  However, there's no harm in assigning the result of `cls.classNameToSignature` to a `const char *` variable as well.